### PR TITLE
Added tutorials for KeplerTargetPixelFile and KeplerLightCurve

### PR DESCRIPTION
**Pull request by christinahedges**
These are some short but sweet tutorials on how to get started with KeplerTargetPixelFile and KeplerLightCurveFile. They're ported from my exoplanet tutorials, I plan to add more.

I also fixed a small bug with the phase, it was broken.

Let me know what you think!

### CODE REVIEW

**Comment by christinahedges** [comment]
Ah my sym links are no longer acceptable that's a pain. Let me try to fix that.

File: lightkurve/tests/test_lightcurve.py [hunk 1]
```diff
@@ -55,9 +55,14 @@ def test_bitmasking(quality_bitmask, answer):
...
-    assert_almost_equal(lc.fold(period=1).time[0], 0)
-    assert_almost_equal(lc.fold(period=1, phase=-0.1).time[0], 0.1)
+    lcf = KeplerLightCurveFile(TABBY_Q8, quality_bitmask=None)
+    lc = lcf.PDCSAP_FLUX.remove_nans()
+    fold = lc.fold(period=1)
+    assert_almost_equal(fold.time[0], -0.5, 2)
+    fold = lc.fold(period=1, phase=-0.1)
+    assert_almost_equal(np.min(fold.time), -0.5, 2)
+    assert_almost_equal(np.max(fold.time), 0.5, 2)
+    assert_almost_equal(fold.time[np.argmin(fold.flux)], 0.319, 2)
```
Comment (mirca) [inline lightkurve/tests/test_lightcurve.py]: it's not clear to me where  ``0.319`` came from.

I quite like the simple test before, i.e., it was clear to me that 
``lc = LightCurve(time=[1, 2, 3], flux=[1, 1, 1])``.

Is it possible to, in addition to this test you've added, to keep the old one, but correct it according to the changes you've made?

**Comment by mirca** [comment]
In the ``tpf`` tutorial, I suggest the follwing:
 - rather than ``from lightkurve import targetpixelfile``, let's do ``from lightkurve import KeplerTargetPixelFile``. That's more intuitive and short and it's also the ``astropy`` practice.

**Comment by mirca** [comment]
Also, in the ``lc`` tutorial:
```
from lightkurve import lightcurve ---> from lightkurve import KeplerLightCurveFile
```

File: lightkurve/tests/test_lightcurve.py (hunk 1 above)
Comment (mirca) [reply lightkurve/tests/test_lightcurve.py]: Let's keep the old test as well. It does not hurt, and it's more intuitive.

Comment (mirca) [reply lightkurve/tests/test_lightcurve.py]: Also, can you add this note you've written about ``0.319`` as a comment in the code?

Comment (christinahedges) [reply lightkurve/tests/test_lightcurve.py]: I'm actually removing the test I've changed my mind I don't like it.

File: lightkurve/tests/test_lightcurve.py [hunk 2]
```diff
@@ -56,8 +56,15 @@ def test_bitmasking(quality_bitmask, answer):
 def test_lightcurve_fold():
     """Test the ``LightCurve.fold()`` method."""
     lc = LightCurve(time=[1, 2, 3], flux=[1, 1, 1])
-    assert_almost_equal(lc.fold(period=1).time[0], 0)
-    assert_almost_equal(lc.fold(period=1, phase=-0.1).time[0], 0.1)
+    assert_almost_equal(lc.fold(period=1).time[0], -0.5)
+
```
Comment (mirca) [inline lightkurve/tests/test_lightcurve.py]: how about this line ``assert_almost_equal(lc.fold(period=1, phase=-0.1).time[0], 0.1)``?
Could you add it back, but obviously with the expected result according to your modifications?

**Comment by mirca** [comment]
Maybe I'm not understanding the changes in ``fold`` done by this PR.

Before:
screen shot 2018-01-28 at 4 52 52 pm

After:
screen shot 2018-01-28 at 4 51 34 pm

File: lightkurve/lightcurve.py [hunk 3]
```diff
@@ -129,7 +129,9 @@ def fold(self, period, phase=0.):
             A new ``LightCurve`` in which the data are folded and sorted by
             phase.
         """
-        fold_time = (((self.time - (phase * period)) / period) % 1) - 0.5
+        fold_time = (((self.time - ((phase) * period)) / period) % 1)
+        #Fold at -0.5 to 0.5 instead of 0 to 1
+        fold_time[fold_time>0.5]-=1
```
Comment (mirca) [inline lightkurve/lightcurve.py]: add white spaces before and after math operators

File: lightkurve/tests/test_lightcurve.py [hunk 4]
```diff
@@ -55,14 +55,13 @@ def test_bitmasking(quality_bitmask, answer):
 
 def test_lightcurve_fold():
     """Test the ``LightCurve.fold()`` method."""
-    lc = LightCurve(time=[1, 2, 3], flux=[1, 1, 1])
-    assert_almost_equal(lc.fold(period=1).time[0], -0.5)
-
-    lcf = KeplerLightCurveFile(TABBY_Q8, quality_bitmask=None)
-    lc = lcf.PDCSAP_FLUX.remove_nans()
+    lc = LightCurve(time=np.linspace(0,10,100), flux=np.zeros(100)+1)
```
Comment (mirca) [inline lightkurve/tests/test_lightcurve.py]: add white spaces after comma

File: lightkurve/lightcurve.py [hunk 5]
```diff
@@ -129,7 +129,9 @@ def fold(self, period, phase=0.):
             A new ``LightCurve`` in which the data are folded and sorted by
             phase.
         """
-        fold_time = ((self.time - phase + 0.5 * period) / period) % 1 - 0.5
+        fold_time = (((self.time - ((phase) * period)) / period) % 1)
```
Comment (mirca) [inline lightkurve/lightcurve.py]: also, no need for parenthesis between ``phase``, i.e.,
``(((self.time - ((phase) * period)) / period) % 1) ---> (((self.time - phase * period) / period) % 1) ``

**Comment by barentsen** [comment]
Let's remove these:
```
%load_ext autoreload
%autoreload 2
```

**Comment by barentsen** [comment]
This is exactly what we need!  Thanks @christinahedges !

**Comment by barentsen** [comment]
Merging into master -- @mirca is going to fix the tiny issues outstanding directly on master* will I look at the next PR!

(* because we're still figuring out why pushing to this PR branch isn't allowed)

**Pull request merged**